src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,42 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Driver olga = new Driver();
+        olga.setName("Olga");
+        olga.setLicenseNumber("5678");
+        olga.setId(10L);
```

Comment:
you don't need to set the id for the driver. This id will be set up automatically after creating row in DB